middleware.ts

```diff
@@ -8,13 +8,40 @@ import { pathToRegexp } from "path-to-regexp";
 // see https://github.com/open-sauced/insights/pull/1549
 // prettier-ignore
 const pathsToMatch = [
+  "/",
   "/hub/insights/:path*",
   "/feed/",
   "/user/notifications",
   "/user/settings",
   "/account-deleted"
 ];
 
+const NO_ONBOARDING_PAYLOAD = {
```

Comment:
Similar to what's in https://github.com/open-sauced/app/blob/d7a69fd8c648892d80a51112730226091b544b10/lib/hooks/useSession.ts#L14-L30